test/proposal.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Arc, DevArcConfig as arcConfig, ProposalData, Proposal } from "../src";
+import { render, screen } from "@testing-library/react";
+
+describe("Proposal component ", () => {
+  it("Shows proposal id", async () => {
+    const proposalId =
+      "0x6afee092a28c74f6358093d5376ac75014ac4d9fd42d296a5498ef42eecd7248";
```

Comment:
One thing this is making me aware of is that when we update to the new client, we'll be using the test_env docker images which have their own set of addresses & proposal IDs. So all of these tests will need to be updated when we do that. Here's the test_env I'm talking about: https://github.com/daostack/test_env

Reply:
Yup, I thought about that, that's why I created a variable for every id so we can change it easier